zebrad/src/config.rs

```diff
@@ -105,6 +105,10 @@ pub struct TracingSection {
     /// If you provide a path with an extension the extension will be ignored and
     /// replaced with `.folded` and `.svg` for the respective files.
     pub flamegraph: Option<PathBuf>,
+
+    /// The journald flag is used for activating of sending tracing events to
```

Comment:
Oops I think I was in a bit of a hurry with my earlier suggestions!
```suggestion
    /// The journald flag is used to send tracing events to
```